domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -72,18 +72,23 @@ class CameraXCameraUseCase @Inject constructor(
 
     //TODO apply flash from settings
     private val imageCaptureUseCase = ImageCapture.Builder().build()
-    private val previewUseCase = Preview.Builder().build()
+    private val previewUseCase: Preview = Preview.Builder().setResolutionSelector(
+        ResolutionSelector.Builder()
+            .setResolutionStrategy(ResolutionStrategy.HIGHEST_AVAILABLE_STRATEGY).build()
+    ).build()
 
     private val recorder = Recorder.Builder().setExecutor(defaultDispatcher.asExecutor()).build()
     private val videoCaptureUseCase = VideoCapture.withOutput(recorder)
     private var recording: Recording? = null
 
     private lateinit var useCaseGroup: UseCaseGroup
 
-    private lateinit var aspectRatio : AspectRatio
+    private lateinit var aspectRatio: AspectRatio
     private var singleStreamCaptureEnabled = false
     private var isFrontFacing = true
 
+    private var currentSurfaceProvider: SurfaceProvider? = null
```

Comment:
Why do we need to hold on to this separately?

Reply:
oops I was trying something else. Forgot to revert this